README.md

```diff
@@ -20,12 +20,14 @@ Install CyFi with pip:
 pip install cyfi
 ```
 
-If you're on a Mac, you should install from conda-forge instead:
+Alternatively, CyFi can be installed with conda:
 
 ```
-conda install -c conda-forge cyfi -y
+conda install -c conda-forge cyfi
 ```
 
+There is a known issue with the pip installation on M1 Macs due to LightGBM. If you encounter this, we recommend installing CyFi with conda.
```

Comment:
You might've missed my last comment: 
https://github.com/drivendataorg/cyfi/pull/134#discussion_r1443570032

I think it would be useful to add a link to the issue at least. 

It might also be nice to use the fancy GitHub blockquotes (although they'll just render as a normal blockquote on the PyPI version of the README) but I don't have a strong feeling about this: https://gist.github.com/jayqi/de4fbef3082b7cebead384be365485fb